src/components/v2/MyAccount/MyAccount.tsx

```diff
@@ -0,0 +1,106 @@
+/** @jsxImportSource @emotion/react */
+import React, { useState } from 'react';
+import Box from '@mui/material/Box';
+import Paper from '@mui/material/Paper';
+import { ISliderProps, Slider } from '../Slider';
+import { useMyAccountStyles as useStyles } from './MyAccountStyles';
+import { Toggle } from '../Toggle';
+import { Icon } from '../Icon';
+import { Tooltip } from '../Tooltip';
+
+const myAccountResources = {
+  myAccountTitle: 'My account',
+  apyWithXvs: 'APY with XVS',
+  apyWithXvsTooltip: 'tooltip content',
+  netApy: 'Net APY',
+  dailyEarnings: 'Daily earnings',
+  supplyBalance: 'Supply balance',
+  borrowBalance: 'Borrow balance',
+  borrowLimit: 'Borrow limit:',
+  safeLimit: 'Your safe limit:',
+};
+
+interface IMyAccountProps {
+  netApy: string;
+  dailyEarnings: string;
+  supplyBalance: string;
+  borrowBalance: string;
+  borrowLimit: string;
+  borrowLimitSum: string;
+  safeLimit: string;
+  sliderProps: ISliderProps;
```

Comment:
We'll want to pass the actual values to the component, so `BigNumber` values instead of strings.
These values will then be used to render the slider, so we shouldn't need to pass `sliderProps` at all.

Reply:
shouldn't we in order of dumb components calculate inside of parent hook?

Reply:
Calculations are fine inside a dumb component. We call them dumb because they work as a black box: if you always pass the same props to a dumb component, it will always render the same thing.
The container will be in charge of fetching this data and handling any other side effect needed.

Reply:
requested calculations description from Sean.
I can make it in separate task in order to merge this one. what do you think of this?

Reply:
yep. but there should be extra wrappers for mapping to format that awaits the frontend like `${formatToDollars(dailyEarningsCents)}`

Reply:
okay. I'll use numbers format for props in this component. but still need calculations for passing into the slider component. so can't do it without formulas from thread in slack.
so we can merge this one in order to add calculations within the next task

added TODO here

Reply:
About the `formatToDollars`, you can create this util in this PR :) It will be relevant to a lot of other parts of the app too.

I'm sorry but what calculations are you thinking about?
The slider only displays the current borrow balance, the safe limit and the borrow limit. You can then use these values to define the props that get passed to the slider:
```
const min = 0;
const max = borrowLimitCents;
const mark = Math.round(borrowLimitCents * safeLimitPercentage / 100);

<Slider min={min} max={max} mark={mark}  />
```

Is there something missing here?

Reply:
there can be also a react component, I think:

https://github.com/VenusProtocol/venus-protocol-interface/pull/244/files#diff-6fcef2bd92cbf1c6b895efaa8c8eda1c978cd54f7a5f4808037c2ba75d0658f7R25

so we just pass into dumb component already prepared strings/components. and it allows us to make more clear atoms and molecules. all calculations and formatting will be made in parent container.

Reply:
Why would `borrowBalance` be a component? We don't need to make the `MyAccount` component ultra modular, it only needs to have just enough flexibility to render exactly like in the designs.

And about the calculations, again there's no calculations there. We only pass the values to the component, which then renders them.
If you're referring to how we're fetching and calculating the values that are passed to `MyAccount`, that will indeed be done in its parent container, but you can avoid having `sliderProps` as a prop on `MyAccount`.

Reply:
yeah. you are right about modularity. 
I thought about separate layers for ui and for side effects in order to split business logic from ui

Reply:
fixed UI props